packages/discovery-provider/plugins/pedalboard/packages/storage/src/index.ts

```diff
@@ -987,6 +987,7 @@ export type Tracks = {
   is_download_gated: boolean;
   download_conditions: unknown | null;
   is_original_available: boolean;
+  playlists_containing_track: number[] | null;
```

Comment:
@alecsavvy wasn't sure about this one. Reed mentioned these may be auto-generated?